src/IIIFPresentation/Core/Web/TypedPathTemplateOptions.cs

```diff
@@ -0,0 +1,54 @@
+namespace Core.Web;
+
+/// <summary>
+/// A collection of options related to path generation. This differs from <see cref="PathTemplateOptions"/> as this
+/// models objects with differing templates depending on 'type' parameter.
+/// </summary>
+public class TypedPathTemplateOptions
+{
+    public const string SettingsName = "PathRules";
+
+    internal static readonly Dictionary<string, string> DefaultFormats = new ()
+    {
+        ["ManifestPrivate"] = "{customerId}/manifests/{resourceId}",
+        ["CollectionPrivate"] = "{customerId}/collections/{resourceId}",
+        ["ResourcePublic"] = "{customerId}/{hierarchyPath}",
+        ["Canvas"] = "{customerId}/canvases/{resourceId}"
```

Comment:
I think these should all start with a `/`? In `HttpRequestX` there's logic to check if the path starts with `/`, and it adds it if it doesn't. 

```cs
if (!string.IsNullOrEmpty(path) && !path.StartsWith(PathDelimiter))
{
  path = PathDelimiter + path;
}
```

If the defaults don't have `/` as a prefix then _every_ default request will do some string manipulation to add it when it seems easier to just add that to the template, particularly as this code is in hotpath and will be called a lot? Alternatively use `/` for all path values and avoid the need to ensure it's there in `GetDisplayUrl()`?